docs/src/Button.doc.js

```diff
@@ -295,8 +301,7 @@ function MenuButtonExample() {
         <Button
           accessibilityExpanded={!!open}
           accessibilityHaspopup
-          color="white"
-          iconEnd="arrow-down"
+          iconEnd={open ? "arrow-down" : "arrow-forward"}
```

Comment:
@AlbertCarreras could we revert the dropdown icon change? Instead, we should toggle the `selected` state when the dropdown is open (please verify with design)

Reply:
DONE.